### ISSUE TITLE
"no-unused-expression" rule now flags "use-strict" as a violation

### ISSUE THREAD

**Issue by HamletDRC** [palantir/tslint#1215]
### Bug Report 

- __TSLint version__: `3.9.0`
- __TypeScript version__: `1.8.10`
- __Running TSLint via__: grunt-tslint 3.1.0

#### TypeScript code being linted

```ts
'use strict';
class MyClass {
}

export = MyClass;
```

with `tslint.json`:

```json
"no-unused-expression": true
```

#### Actual behavior

>> (no-unused-expression) src/navigation/GoToUrlAction.ts[3, 1]: expected an assignment or function call

#### Expected behavior

use strict should not trigger this. 

Perhaps my use-strict is in the wrong location within this file. Is it? 
Regardless, it is not an unused expression. It is an ignored literal at best.

**Comment by HamletDRC** [comment]
i have some time to send a pull request for this... i just want to make sure you want it fixed in tslint and it is not some silly mistake on my side.

**Comment by adidahiya** [comment]
haven't verified this as a bug yet, but it is a little surprising given the existing test cases https://github.com/palantir/tslint/blob/master/test/rules/no-unused-expression/test.ts.lint

that said, if you can generate a failing test case and send a PR to fix it, that would be great!

**Comment by JKillian** [comment]
As a sidenote, [please note the valid locations for JS directives](https://tc39.github.io/ecma262/#sec-directive-prologues-and-the-use-strict-directive), these are the only places we should allow standalone string literals.

I ran TSLint on your code and didn't get the same error, so it's also possible that `grunt-tslint` is using an older version of TSLint?

**Comment by HamletDRC** [comment]
I updated my tslint dependency and the false-positive disappeared.
